specification/terraform/Microsoft.AzureTerraform.Management/models.tsp

```diff
@@ -92,45 +92,51 @@ model ExportResourceGroup extends BaseExportModel {
   type: "ExportResourceGroup";
 }
 
+// Not using Azure.ResourceManager.ArmOperationStatus due to bug in the API response where id is a long URL instead of uuid
+// Will be fixed at next API version
 @doc("The status of the LRO operation.")
-model OperationStatus {
-  @doc("The operation status resource id.")
-  id?: string;
+model TerraformArmOperationStatus<
```

Comment:
how about we name this model as `ArmOperationStatus`? because your comment says in your next version, this should be replaced by ArmOperationStatus

Reply:
@ArcturusZhang after consulting with @magodo : the `id` property of the 202 response from GET /operationStatuses is a RPaaS feature, not a bug. It's a surprise to us that `ArmOperationStatus` does not conform to this.

Instead I have changed this comment to indicate why we cannot use ArmOperationStatus. See: 4845ff64e0.

Reply:
@gerrytan, @ArcturusZhang - Can you follow up on the Typespec Discussion channel (https://teams.microsoft.com/l/channel/19%3A906c1efbbec54dc8949ac736633e6bdf%40thread.skype/TypeSpec%20Discussion?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) to let the owners know about this discrepancy and figure out a fix/improvement?

Reply:
Thx @mentat9 , I posted this problem [here](https://teams.microsoft.com/l/message/19:906c1efbbec54dc8949ac736633e6bdf@thread.skype/1729469943546?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1729469943546&teamName=Azure%20SDK&channelName=TypeSpec%20Discussion&createdTime=1729469943546).

Meanwhile I've been able to get around this by not declaring `getOperationStatus` operation directly (ie: just declare it as part of `exportTerraform` LRO headers)